benchmarks/model_benchmarks/model_validate/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from django.core.exceptions  import ValidationError
+from django.utils.translation import gettext_lazy as _
+from ...utils import bench_setup
+
+bench_setup()
+
+def validate_name(name):
+    if name != 'abc':
+        raise ValidationError(
+            _('%(name)s is not abc'),
```

Comment:
Do we need the translation of this string? 

I think we're trying to to benchmark the "validate" so should we strip it down to the core "happy path"?

Reply:
I will remove the string translation.